stl/src/xonce2.cpp

```diff
@@ -24,4 +26,22 @@ int __stdcall __std_init_once_complete_clr(void** _LpInitOnce, unsigned long _Dw
     return InitOnceComplete(reinterpret_cast<LPINIT_ONCE>(_LpInitOnce), _DwFlags, _LpContext);
 }
 
+[[noreturn]] void __stdcall __std_init_once_link_alternate_names_and_abort() noexcept {
+    _CSTD abort();
+}
+
+#if defined(_M_ARM64EC)
+#error ARM64EC is not expected here
+#elif defined(_M_HYBRID)
+#error CHPE is not expected here

```

Comment:
I think we need to tolerate and ignore ARM64EC and CHPE here. Both platforms presumably need the `_clr` APIs defined above, as well as `__std_init_once_link_alternate_names_and_abort` (although we could easily workaround that particular need in the header), so the internal build is attempting to compile this TU to include in the static/import libs for those architectures and failing.

Reply:
Yep, you're right. I'm going to drop this from the current merge batch and rework it as you suggest.